packages/custom_lint/test/cli_test.dart

```diff
@@ -86,6 +87,61 @@ lib/custom_lint_client.dart:13:29: Error: Undefined name 'createPlugin'.
     );
   });
 
+  test('exits with 0 when pass argument `--no-fatal-infos`', () async {
+    final plugin = createPlugin(name: 'test_lint', main: helloWordPluginSource);
+
+    final app = createLintUsage(
+      source: {
+        'lib/main.dart': 'void fn() {}',
+      },
+      plugins: {'test_lint': plugin.uri},
+      name: 'test_app',
+    );
+
+    await runWithIOOverride(
+      (out, err) async {
+        await cli.entrypoint(['--no-fatal-infos']);
+
+        expect(err, emitsDone);
+        expect(exitCode, 0);
```

Comment:
Could you test that `out` still logs the infos/warnings?

Reply:
fixed commit 882b9bb